src/client/dfuse/dfuse_core.c

```diff
@@ -674,7 +678,7 @@ dfuse_set_default_cont_cache_values(struct dfuse_cont *dfc)
 	dfc->dfc_dentry_timeout     = 1;
 	dfc->dfc_dentry_dir_timeout = 5;
 	dfc->dfc_ndentry_timeout    = 1;
-	dfc->dfc_data_caching       = true;
+	dfc->dfc_data_timeout       = 60 * 10;
```

Comment:
curious, why 600 here?

Reply:
We've got a draw a line somewhere, I wanted something that would help and be meaningful for many use-cases but that wasn't forever.